pyproject.toml

```diff
@@ -71,6 +71,11 @@ ruff = "0.5.5"
 yamllint = "*"
 markdown-include = "*"
 toml = "*"
+# Python implementation of markdownlint
+pymarkdownlnt = [
+    {version = "0.9.26", python = "~3.8.0"},
```

Comment:
Why do we care about 3.8? That's EOL and no longer supported.

Reply:
That should be removed. This was for projects that still support 3.8 since the cookie hasn't been updated to drop 3.8 support yet.

Reply:
Made the update myself so we can get this merged in.

Reply:
Thank you!